3rdparty/bazel/com_github_google_go_tpm_tools/com_github_google_go_tpm_tools.need

```diff
@@ -8,10 +8,7 @@ go_library(
         "internal_cross.go",
     ],
     cgo = True,
-    cdeps = select({
-        "@//bazel/settings:tpm_simulator_enabled": ["//simulator/ms-tpm-20-ref:ms_tpm_20_ref"],
-        "//conditions:default": ["@//3rdparty/bazel/com_github_google_go_tpm_tools/placeholder:ms_tpm_20_ref_disabled"],
-    }),
+    cdeps = cdeps = ["//simulator/ms-tpm-20-ref:ms_tpm_20_ref"],
```

Comment:
This looks weird - is that intentional?

Reply:
The reason this is necessary is that when using bzlmod, other repos (like `@com_github_google_go_tpm_tools` cannot see the root repo (`@//` or `@constellation`).

In practice, this method for disabling the simulator is not needed, since our Go code also has Go build tags surrounding it's usage (search for `tpm_simulator_enabled`).

Reply:
Will fix